src/mpid/ch4/shm/posix/eager/fbox/fbox_init.c

```diff
@@ -30,7 +30,7 @@
 
 #define MPIDI_POSIX_MAILBOX_INDEX(sender, receiver) ((MPIDI_POSIX_global.num_local) * (sender) + (receiver))
 
-int MPIDI_POSIX_eager_init(int rank, int size)
+int MPIDI_POSIX_fbox_init(int rank, int size)
 {
```

Comment:
The commit message does not explain why you need change the func name. I like the renaming. But perhaps it should not be in this commit. How about move it to the fbox uninline commit? Do you need the same renaming for stub?

Reply:
I will move the name change and fix the commit message. Concerning stub, as I understand it these are there only as blueprint for developers that want to add new modules; So probably I should also change these although it does not affect functionality.